io/src/main/scala/fs2/io/file/pulls.scala

```diff
@@ -22,6 +23,22 @@ object pulls {
       h: FileHandle[F]): Pull[F, Byte, Unit] =
     _readRangeFromFileHandle0(chunkSize, start, end)(h)
 
+  def keepReadingFromFileHandle[F[_]: Timer](chunkSize: Int, delay: FiniteDuration)(
```

Comment:
Name wise, how about `tailFileHandle`? Also, I think we should have a convenience function for this in `file` package object like `fs2.io.file.tail(...)`

Reply:
Right. So, 
- Do you think `file.tail(...)` should ignore preexisting content of the file at the time it's run?
- Should there be a parameter saying how far _before_ the end of file the output should start, as in `tail -50f ...` (but probably in bytes rather than lines)?
- Is there any chance starting to read a text file at some arbitrary position would mess up UTF decoding?
(I didn't use the name `tail`, because, as is, the function reads the file from the beginning and only then tails it.)

Reply:
Great questions.

```scala
def tail[F[_]: Sync: ContextShift: Timer](path: Path, blocker: Blocker, chunkSize: Int, offset: Long = 0L, pollDelay: FiniteDuration = 1.second): Stream[F, Byte] = ???
```

The `offset` param is an offset from the *start* of the file, not the end.

Thoughts?

Reply:
My concern was that since UTF-8 is variable width, starting at an arbitrary position, which could be in the middle of a rune, could maybe mess up UTF decoding. However, I looked up UTF-8 and since the prefixes of first and subsequent bytes in multi-byte encoded characters are different, there won't be any ambiguity for decoders. So, it's fine.

Reply:
Done.